node/Dockerfile

```diff
@@ -33,6 +33,10 @@ FROM clearlinux/os-core:latest
 
 COPY --from=builder /install_root /
 
+ENV NAME="node" \
+    VERSION="v12.13.0" \
```

Comment:
In Clear, the version is changing from time to time. How can you make sure it is always v12.13.0?
And if no automatic way, how can you make sure your manual update is always in time?